tests/unit_tests/test_wave_06.py

```diff
@@ -109,8 +108,16 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
+    assert result is True
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert item_f in tai.inventory 
+    assert item_c in jesse.inventory 
+    assert item_a in tai.inventory
+    assert item_d in jesse.inventory
```

Comment:
Because we have written the code, we know what jesse's inventory and tai's inventory should look like after a swap has been made. Instead of checking to see if each item is in its respective inventory, what would happen if we created lists that hold the expected items and then check for equality?